src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -199,8 +202,8 @@ def __init__(self, run: Run, stream=False, run_mode=RunMode.Test):
         # for line run records, store per line
         # for normal node run records, store per node per line;
         # for reduce node run records, store centralized in 000000000.jsonl per node
-        outputs_folder = self._prepare_folder(self.path / "flow_outputs")
-        self._outputs_path = outputs_folder / "output.jsonl"
+        self.outputs_folder = self._prepare_folder(self.path / "flow_outputs")
```

Comment:
`self._output_folder`?

Reply:
We use it in `run_submitter`, it is not a private variable.
![image](https://github.com/microsoft/promptflow/assets/111329184/9f83d748-e987-4dfb-a356-aac1abff112a)